segwit2x-annuncement.md

```diff
@@ -0,0 +1,50 @@
+# Bitcoin Upgrade at Block 494,784
+
+During the month of November 2017, approximately 90 days after the activation of Segregated Witnesses in the Bitcoin blockchain, a block between 1MB and 2MB in size will be generated by Bitcoin miners in a move to increase network capacity. At this point it is expected that more than 90% of the computational capacity that secures the Bitcoin network will carry on mining on top of this large block.
+
+The upgrade to 2MB blocks has been agreed first during the [Bitcoin Roundtable Consensus in Hong-Kong](https://medium.com/@bitcoinroundtable/bitcoin-roundtable-consensus-266d475a61ff) on February 2016, and then ratified by the [Bitcoin Scaling Agreement in New York](https://medium.com/@DCGco/bitcoin-scaling-agreement-at-consensus-2017-133521fe9a77) on May 2017. These agreements stipulate the activation of Segregated Witness support and an increase of the maximum block size from 1MB to 2MB.
+
+Segregated Witness support has been locked-in as a soft fork expected to activate around August 23 2017 (block height 481,824). Bitcoin clients that are not currently SegWit-compatible and wish to benefit from the new type of transaction must perform extensive upgrades to various subsystems, including changes to transaction serialization, signature hash computation, block weight calculation, scripting engine, block validation, a new address scheme, and P2P protocol upgrades. Fortunately Segregated Witness compatibility is opt-in, and existing Simplifed Payment Verification (SPV) wallets and full nodes are expected to continue working without changes after SegWit activates.
+
+## Readiness Checklist
+
+The November 2107 upgrade to 2MB blocks is a hard-fork, but necessary changes are trivial to perform. Some SPV clients are expected to work without any change at all. Most clients will need to tweak only two constants to remain compatible with the new larger blocks:
```

Comment:
November 2107 may be safer timing for this hard fork from the standpoint of users and developers concerned with network stability, but the New York Agreement wants to fork this year, so please change this text to show the aggressive timeline accurately as November 2017.